src/map/entities/battleentity.h

```diff
@@ -143,7 +143,8 @@ enum SKILLTYPE
 	SKILL_ALC			= 55,
 	SKILL_COK			= 56,
 	SKILL_SYN			= 57,
-	SKILL_RID			= 58
+	SKILL_RID			= 58,
+	SKILL_DIG			= 59 // Chocobo digging

```

Comment:
I suppose it doesn't really matter since neither display in game, but you sure those aren't really the same skill? 

I remember ages ago (early 2004 ish it got changed) if you opened the menu fast enough you could see hidden skills like riding and goldfish scooping (and several "blank" slots between them) and then they'd get hidden, back before SE patched the client to not have those menu text strings anymore. IIRC, choco skill was the very last menu entry back when it existed.